### PR TITLE
transfer es-ui-upgrades to CromhaPlugins

### PR DESCRIPTION
**Pull request by MCOfficer**
As per https://github.com/lumbar527/es-ui-upgrades/commit/841bd780a8e8499670b83d120cdc7c70f174b214

@OcelotWalrus @lumbar527 please approve

### CODE REVIEW

File: generated/plugins.yaml [hunk 1]
```diff
@@ -412,20 +412,20 @@
     type: tag
     url: https://github.com/beccabunny/Beccas-Endless-Expansion/archive/refs/tags/$version.zip
 - name: ES-ui-upgrades
-  authors: Roger Tabris
-  homepage: https://github.com/lumbar527/es-ui-upgrades
+  authors: '@OcelotWalrus/Cromha'
```
Comment (lumbar527) [inline generated/plugins.yaml:415]:
```suggestion
  authors: Roger Tabris
```
Again

File: manifests/es-ui-upgrades.yaml [hunk 2]
```diff
@@ -1,15 +1,15 @@
 name: ES-ui-upgrades
-authors: Roger Tabris
-homepage: https://github.com/lumbar527/es-ui-upgrades
+authors: '@OcelotWalrus/Cromha'
```
Comment (lumbar527) [inline manifests/es-ui-upgrades.yaml:2]:
```suggestion
authors: Roger Tabris
```

**Review by lumbar527 — APPROVED** [review]
Pretty good except the author change.

**Review by OcelotWalrus — APPROVED** [review]
I approve

**Pull request merged**